applications/app/views/fragments/paidAdvertisementGalleryBody.scala.html

```diff
@@ -75,7 +75,11 @@
 
     @if(!page.gallery.content.shouldHideAdverts && rowNum % adInterval == 0) {
         <li class="gallery2__item gallery2__item--advert">
-            <div class="gallery2__img-container js-gallery-slot">
+            <div class="gallery2__img-container">
+                @defining(rowNum / adInterval) { inlineId =>
+                    @gallerySlot(inlineId)
+                    @gallerySlot(inlineId - 1, isMobile = true)
+                }
             </div>
         </li>
     }
```

Comment:
Is there any possibility of merging this template with `galleryBody` or are they very different?
I know it's outside the scope of this PR.